### PR TITLE
Support windows

### PR DESCRIPTION
**Pull request by tamago324**
Hi @kristijanhusak 
Thank you for the great plugin ✨

It now works on Windows.

I've tested all three on Ubuntu and Windows.

* PostgreSQL
* MySQL
* SQL Server

However, the connection from Neovim on Windows to PostgreSQL could not be made because of these [issue](https://github.com/tpope/vim-dadbod/issues/63). Maybe it's just my environment.

I have only checked `omnifunc`, should I also check `coc.nvim` and `deoplete.nvim`?
If I have to check, I will check.

Thank you!

### CODE REVIEW

**Comment by kristijanhusak** [comment]
@tamago324 I left one comment on code. Please review it so we can wra0 up this support.

**Comment by tamago324** [comment]
Thank you for your comment.
I'm sorry. Where is the comment on the code? I don't know where that comment is located.

File: autoload/vim_dadbod_completion.vim [hunk 1]
```diff
@@ -163,7 +163,8 @@ function! s:generate_query(db, query_key, ...) abort
   if a:0 > 0
     let Query = Query(a:1)
   endif
-  return printf('%s %s', base_query, Query)
+  " for PostgreSQL
+  return printf('%s %s', substitute(base_query, '"', '', 'g'), Query)
```
Comment (kristijanhusak) [inline autoload/vim_dadbod_completion.vim:167]: Why this has to be done? Isn't dadbod taking care of this?

**Comment by kristijanhusak** [comment]
@tamago324 https://github.com/kristijanhusak/vim-dadbod-completion/pull/5/files#r422872434

File: autoload/vim_dadbod_completion.vim (hunk 1 above)
Comment (tamago324) [reply autoload/vim_dadbod_completion.vim]: A string like the following was returned

```
psql -w --dbname "postgresql://postgres:postgres@127.0.0.1:5432/shop"
```

However, it was confirmed that it works normally even if you don't replace it with `substitute()`.
Therefore, it is removed.

**Comment by kristijanhusak** [comment]
Thanks!

**Comment by tamago324** [comment]
Thank you so much for merging!

**Pull request merged**